bin/updateVersion.mjs

```diff
@@ -0,0 +1,120 @@
+#! /usr/bin/env node
+import semver from 'semver'
+import { exec } from 'child_process'
+import { readFile, writeFile } from 'fs/promises'
+
+const REPO_URL = 'https://github.com/nrkno/Sofie-TV-automation'
```

Comment:
If am I am reading the purpose of this variable correctly, it is a default repo to fall back on if there is no repo specified in th package.json.

Would it be an idea to stop the execution of the script instead of defaulting to your repository?

Usecase: We run the script in our repo. For some weird reason we don't have a repo specified in our package.json, so this script will start executing on your repository. 

If the above is the actually behavoiur (not sure if it is, please correct me if I am wrong :) ) then we (or some other third party) would potentially run the script thinking we are updating our versions, but accidentally updating yours?

Reply:
I agree, it would be better to abort execution if there is no usable url in package.json, rather than generate the changelog with a guaranteed wrong url